### PR TITLE
Happy Thoughts by Erika

### PR DESCRIPTION
**Pull request by Tejpex**
## Netlify link
[https://boisterous-brioche-b12b23.netlify.app/]

## Collaborators
Add your collaborators here. Write their GitHub usernames in square brackets. If there's more than one, separate them with a comma, like this:
[github-username-1, github-username-2]

### CODE REVIEW

**Review by dzc1 — APPROVED** [review]
Hey Erika!! 

The project is looking great, all requirements have been met and the syntax as well as logic is looking great. Hope that you are enjoying the power of React. 

Keep up the amazing work!

File: src/index.css [hunk 1]
```diff
@@ -1,13 +1,201 @@
...
+}
+
+.thoughts-section {
+  width: 65%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.thought-box {
```
Comment (Lovisaaberg) [inline src/index.css:112]: Sometimes the characters go outside the thought-box, suggestion to maybe add word-wrap: break-word.

**Review by Lovisaaberg — COMMENTED** [review]
The code is easily read, the project has a good structure and is responsive, great job! Nice feature that the color on the like button changes back to its original color when clicked again - though the number of likes still adds. Also just some minor fixes: Check the contrast and the font size on the text for accessibility. Good job with the design! 👏

File: src/index.css (hunk 1 above)
Comment (Tejpex) [reply src/index.css:112]: Thanks, I will try that! :)

**Pull request closed without merging**